contract/.env.example

```diff
@@ -1,2 +0,0 @@
-RPC_URL=
-MNEMONIC=
```

Comment:
This file is necessary to use `HDWalletProvider` for truffle and therefore to deploy contracts.
Could you restore it?